src/Mod/Fem/App/HypothesisPy.cpp

```diff
@@ -1178,7 +1180,7 @@ StdMeshers_MEFISTO_2DPy::StdMeshers_MEFISTO_2DPy(int hypId, int studyId, SMESH_G
     : SMESH_HypothesisPyBase(new StdMeshers_MEFISTO_2D(hypId, studyId, gen))
 {}
 #endif
-
+#endif
 StdMeshers_MEFISTO_2DPy::~StdMeshers_MEFISTO_2DPy() = default;
 
```

Comment:
This endif is misplaced and needs to be one line below.

Reply:
Are you submitting a PR to adjust?

Reply:
#12268